### PR TITLE
Add metagenomics documentation

### PR DESCRIPTION
**Pull request by bmesuere**
- [x] Add framework of files and routes
- [x] Move case study and add redirect from old location
- [x] Add installation instructions
- [x] Add documentation

### CODE REVIEW

**Comment by bmesuere** [comment]
@ninewise Do you have additional content I can put on these pages?

**Comment by ninewise** [comment]
The installation instructions and basic usage are [in the README](https://github.com/unipept/umgap). More usage details are in the `--help` of the commands, but I'll add those, along with small examples.

**Comment by ninewise** [comment]
@bmesuere pinging you here to ask for a review, since you made the PR.

I've not yet rewritten the case study, but I'll do this on another branch once this is merged.

File: app/controllers/api/umgap_controller.rb [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+  end
+
+  def casestudy_metagenomics
+    @title = 'Case study: metagenomics'
+    @sidebar_name = 'Case studies'
+    @study = 'Metagenomics'
+  end
+
+  def fastq2fasta
+    @title = 'The umgap  fastq2fasta command'
```
Comment (bmesuere) [inline app/controllers/api/umgap_controller.rb:25]: There is a double space after umgap in each of these titles

File: app/views/api/umgap/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,144 @@
+<div class="apidocs row">
+  <div class="col-xs-9">
+    <h1 class='api-title'>
+      <span class="initialism">UMGAP</span> documentation
+    </h1>
+    <p class='lead'>
+      Use the Unipept Metagenomics Analysis Pipeline (<span class='initialism'>UMGAP</span>) to analyse and present your findings using frequency tables and interactive visualizations.
```
Comment (bmesuere) [inline app/views/api/umgap/index.html.erb:7]: The introduction lacks the specification of which data this is about. The lead here doesn't even mention data. The next one only talks about metagenomics analyses. (short) shotgun metagenomics reads should at least be mentioned somewhere.

**Review by bmesuere — COMMENTED** [review]
Looks good 👍  For the examples, I would put the entire command in bold and not only the $.
A case study is definitely needed to make sense of the huge list of commands.

**Comment by ninewise** [comment]
@bmesuere I added a short introduction

**Review by ninewise — APPROVED** [review]

**Pull request merged**